client/send-text.cc

```diff
@@ -175,6 +185,24 @@ int main(int argc, char *argv[]) {
             }
         } while (run_forever && !interrupt_received);
     }
+    else if ( rot_vertical == true) { 
+    // rotation for vertical banner text
+        if (scroll_delay_ms > 0) { 
+           do {
+              display.Fill(bg);
+              for (int s = 0; s < total_len + height && !interrupt_received; ++s) {
```

Comment:
split out total_len into two variables in https://github.com/miloh/flaschen-taschen/commit/12f31108b980527cdc05512421d8adbf8aeb5fb7